langlib/lang.regexp/src/main/ballerina/regexp.bal

```diff
@@ -314,23 +428,15 @@ isolated function getReplacementString(Groups groups, Replacement replacement) r
     return replacement(groups);
 }
 
-# Splits a string into substrings separated by matches of a regular expression.
```

Comment:
Any reason why we took this function to the top?
Better stick to the spec order

Reply:
It was suggested before [[1]](https://github.com/ballerina-platform/ballerina-lang/pull/39096#discussion_r1052099292)

Reply:
What I meant there was to move that specific scenario as the first sample under `Split` examples

Reply:
will do my bad